_pages/team.html

```diff
@@ -58,20 +48,15 @@ <h5>Anuragi Thapliya</h5>
             <h5>Joe Carpenter</h5>
             <p>Joe is a programmer, and has worked on various repositories for the team, such as the dashboard and the website.</p>
         </div>
-        <div>
-            <img src="/assets/img/team/zack-biegel.png">
-            <h5>Zack Biegel</h5>
-            <p>Zack is the photographer and videographer of the team and is heavily involved in outreach.</p>
-        </div>
         <div>
             <img src="/assets/img/team/jack-evans.png">
             <h5>Jack Evans</h5>
             <p>Jack is a mechanic.</p>
         </div>
         <div>
-          <img src="/assets/img/team/andrew-lester.png">
```

Comment:
@JosephCarpenter There were more spaces added to the beginning of the lines to keep consistency with the rest of the document. They're highlighted in slightly darker green.

Reply:
oh, that makes sense

Reply:
Not sure what makes that comment funny... but hey, no judgement

Reply:
Idk, somebody else reacted with that and I just clicked on it because bandwagon

Reply:
I reacted because I thought that was removed and there was an actual change, but it was just spaced over and I thought it was funny that he didn't notice changes